src/Raven.Server/Rachis/RemoteConnection/Rider.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Sparrow.Binary;
+
+namespace Raven.Server.Rachis.RemoteConnection
```

Comment:
Maybe change to `Raven.Server.Rachis.Remote`. We will avoid `RemoteConnection.RemoteConnection` all over the place then.